src/routes/game/components/GameOverDialog.vue

```diff
@@ -52,35 +52,61 @@
           </div>
         </div>
       </template>
+      <p v-if="opponentWantsToRematch">
+        {{ gameStore.opponent.username }} wants to rematch
+      </p>
+      <p v-if="opponentDeclinedRematch">
+        {{ gameStore.opponent.username }} declined rematch and left the game
+      </p>
```

Comment:
Let's add testing selectors that we can use to identify these during testing
```suggestion
      <p v-if="opponentWantsToRematch" data-cy="opponent-wants-rematch">
        {{ gameStore.opponent.username }} wants to rematch
      </p>
      <p v-if="opponentDeclinedRematch" data-cy="opponent-declined-rematch">
        {{ gameStore.opponent.username }} declined rematch and left the game
      </p>
```